packages/samples/headless-react/src/components/relative-date-facet/relative-date-facet.fn.tsx

```diff
@@ -0,0 +1,94 @@
+import {useEffect, useState, FunctionComponent} from 'react';
```

Comment:
Let's update the date-facet config under the `doc-parser` directory to reference the new sample files.